animalfacts.py

```diff
@@ -225,6 +225,7 @@ def check_comment_for_animal(comment, reddit):
     botengine('antelope', '\santelopes?\s', reddit, ANTELOPE_FACTS, comment)
     botengine('armadillo', '\sarmadillos?\s', reddit, ARMADILLO_FACTS, comment)
     botengine('atlantic puffin', '\spuffins?\s', reddit, ATLANTIC_PUFFIN_FACTS, comment)
+    botengine('aye-aye', '\saye-aye?\s', reddit, AYE_AYE_FACTS, comment)
```

Comment:
I'm confused: is the name Aye-Aye or Aye Aye? This has issue #168 implications...